Cargo.toml

```diff
@@ -13,21 +13,21 @@ incremental = false
 overflow-checks = true
 
 [workspace.dependencies]
-neutron-sdk = { package = "neutron-sdk", git = "https://github.com/neutron-org/neutron-sdk", branch = "main" }
+neutron-sdk = { package = "neutron-sdk", git = "https://github.com/neutron-org/neutron-sdk", rev = "33e0cb123a4a91a37eb0688a8fd9b52bce320ed9" }
```

Comment:
the rev needs to be updated to the latest rev of https://github.com/neutron-org/neutron-sdk/pull/147 I suppose

Reply:
Done